gsoc2018-bharat/src/lstm.py

```diff
@@ -0,0 +1,99 @@
+import torch
+import torch.autograd as autograd
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+import numpy as np
+from sklearn.datasets.base import Bunch
+import json
+from gensim.models import FastText
+import logging
+import sys
+import argparse
+
+torch.manual_seed(1)
+
+logging.basicConfig(format='%(levelname)s : %(message)s', level=logging.INFO)
+logging.root.level = logging.INFO
+
+wv = None
+
+def load_model(savedModel):
```

Comment:
instead of declaring `wv` as a global, couldn't you just return it ?

Reply:
I was doing that originally. I wanted to see the effect it would have on the memory usage. The entire model takes up a big chunk of memory itself. It was better to get rid of it as soon as the vectors were mapped.